core/src/trezor/sdcard.py

```diff
@@ -26,13 +31,24 @@ def _deinit_instance(self) -> None:
         sdcard.power_off()
         FilesystemWrapper._INSTANCE = None
 
+    def _try_mount(self) -> None:
+        """Try to mount filesystem, raise appropriate exception."""
+        if not self.mounted:
+            return
+        try:
+            fatfs.mount()
+        except OSError as e:
+            if e.args[0] == fatfs.FR_NO_FILESYSTEM:
+                raise NoFilesystem()
```

Comment:
Considering our rules about exceptions:
> If an exception is thrown with no arguments, the exception class SHOULD be thrown instead of a new object.

I am curious, why do we need to create a new object in this case?

Reply:
we don't. i was under the impression that I need to create an object to properly execute the `__init__` method, but that is not the case, as I just checked.

that btw most likely means that a new object is created under the hood anyway, so the recommendation is useless.

(it is more pythonic to `raise NoFilesystem` though, so maybe we should keep the recommendation for style reasons)

Reply:
this code is now removed